contrib/kube-prometheus/jsonnet/kube-prometheus/alerts/prometheus-operator.libsonnet

```diff
@@ -0,0 +1,50 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prometheus-operator',
+        rules: [
+          {
+            alert: 'PrometheusOperatorAlertmanagerReconcileErrors',
+            expr: |||
+              rate(prometheus_operator_alertmanager_reconcile_errors_total{%(prometheusOperatorSelector)s}[5m]) > 0.01
```

Comment:
I’m fairly certain 1% is going to trigger on any error occurring. Let’s bump this to 10% I’d say, or do a 3% warning and 10% critical.